apps/web/src/components/Table/ApiManagedTable/TableHeader.tsx

```diff
@@ -54,12 +54,30 @@ function TableHeader<T extends Record<string, unknown>>({
   const intl = useIntl();
   const methods = useForm();
 
+  const inputId = "tableSearch";
```

Comment:
I think hard coding the ID here is going to cause problems if you have more than one table.  This might be what Wave is noticing on the Edit User page.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/a34bc60e-60e4-4be0-a992-6f2de34cb2c8)

Reply:
@petertgiles please see: e690c210aa3fed88cc7014e55ef954f374a3239e.

## Screenshots

![Screen Shot 2023-05-09 at 10 43 53](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/c9dd8432-45d5-4fec-83e3-53e333ef2bf6)

![Screen Shot 2023-05-09 at 10 44 10](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/47a85668-b50e-4854-8525-3a438747453c)

Reply:
Nice!  Yeah, seems fixed for me, too.

Reply:
First use of that hook that I've seen.  :sunglasses:

Reply:
Used it for the `SkillPicker` component in https://github.com/GCTC-NTGC/gc-digital-talent/pull/5374. I believe @esizer was the original🕵️!